modules/python3/bindings/src/pymesh.cpp

```diff
@@ -296,7 +299,11 @@ void exposeMesh(pybind11::module& m) {
             return self.addVertex(vertex);
         });
 
+    using MeshSequence = std::vector<std::shared_ptr<Mesh>>;
+    py::bind_vector<MeshSequence>(m, "MeshSequence", py::module_local(false));
```

Comment:
Why module_local(false)? added comment

Reply:
Code is based on this https://github.com/inviwo/inviwo/commit/d0b1ddb4c73231cba9dafe252ded032d28814d39

I don't think that the `module_local(false)` is necessary as pybind will try to determine it itself , but you might have had a reason for including it? I tested to remove it and it seems to run without issues at least.

https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html?highlight=module_local#binding-stl-containers
> When binding STL containers pybind11 considers the types of the container’s elements to decide whether the container should be confined to the local module (via the [Module-local class bindings](https://pybind11.readthedocs.io/en/stable/advanced/classes.html#module-local) feature). If the container element types are anything other than already-bound custom types bound without py::module_local() the container binding will have py::module_local() applied.